contracts/utils/interfaces/IOrderPayable.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IOrderPayable {
+    event PaidForOrder(
+        bytes32 indexed orderId,
+        uint256 expirationTimestamp,
+        address orderSigner,
+        uint256 amount,
+        address sender
+    );
+
+    event Withdrew(address recipient, uint256 amount);
```

Comment:
Same with recipient 🤔

Reply:
Same here, I can't think of a user flow that requires us to get all withdrawals to a specific address. When we need it we'll need all of it.